File: javascript/fibonacci_series.js
let n = parseInt(prompt("Enter the number of terms: "));
let t1 = 0, t2 = 1;

console.log("Fibonacci Series:");
for (let i = 1; i <= n; i++) {
    console.log(t1);
    let nextTerm = t1 + t2;
    t1 = t2;
    t2 = nextTerm;
}

File: java/FibonacciSeries.java
import java.util.Scanner;

public class FibonacciSeries {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number of terms: ");
        int n = scanner.nextInt();
        int t1 = 0, t2 = 1;

        System.out.print("Fibonacci Series: ");
        for (int i = 1; i <= n; i++) {
            System.out.print(t1 + ", ");
            int nextTerm = t1 + t2;
            t1 = t2;
            t2 = nextTerm;
        }
        System.out.println();
    }
}
